src/utils/get-youtube-video-id.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function (url) {
```

Comment:
I think we should add a bit of JSDoc here, so that whenever this function is imported elsewhere, it's a little clearer what it does and what its arguments are for.

```suggestion
/**
 * Generates the markup for a responsive YouTube video embed.
 * @param {string} url URL of the YouTube upload
 * @returns {string} YouTube embed markup
 */
module.exports = function (url) {
```

Reply:
LOVE it!

Reply:
@BenDMyers unfortunately, i can't merge because i updated the function names for clarity. i was using it as `getYouTubeId` instead of the more accurate `getYouTubeEmbed`.

i'm very on board with the change. to proceed would you like to:
* suggest it again for an in-PR "Commit suggestion"
* add a commit to this branch
* open a new branch-to-branch PR
* have me copy/paste the suggestion into the updated code

I'm happy to proceed however you see fit

Reply:
I think if you copy-paste, we should be fine!

Reply:
![image](https://user-images.githubusercontent.com/658360/105230536-551a5b00-5b1a-11eb-82df-56073f66444c.png)
💥